pkgs/games/blackvoxel/default.nix

```
@@ -0,0 +1,53 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, makeDesktopItem
+, imagemagick # for icon for desktop item
+, glew110
+, SDL
+}:
+
+stdenv.mkDerivation rec {
+  pname = "blackvoxel";
+  version = "2.50";
```

```suggestion
  version = "2.5";
```
if the tag is without an extra 0 then we should reflect that

---

If you look at https://www.blackvoxel.com/view.php?node=1541 under "Blackvoxel School Mode" or under "Source Code" the version is 2.50. They also refer to an older version 2.42 for example. So should I follow the official version or the git tag ?

---

it would be so easy if the tags would just follow that versioning...

---

I would suggest just leaving the version as it is, so 2.50, so that if there is an update to 2.51 we're prepared

---

I am not sure if the auto update bot would pick this up and create a PR for it.

---

Ah right. To be sure I checked again https://github.com/Blackvoxel/Blackvoxel/releases/tag/2.5 to see if they mention 2.50 and they don't, same for https://www.blackvoxel.com/view.php?node=1758 ... But the download page https://www.blackvoxel.com/view.php?node=1541 says 2.50 and 2.42 ...
Looking also at "news" at https://www.blackvoxel.com/view.php?node=22 it looks like they like to mix versions. 2.1, 2.20, 2.40, 2.5 ...

I suppose the conclusion is that since they aren't themselves consistent in their versioning on their website, let's just use the github tag and spare ourselves pain.